src/Plugin/TfaValidation/TfaBasicSms.php

```diff
@@ -25,14 +25,12 @@ public function __construct(array $context, $mobile_number) {
     if (!empty($context['validate_context']) && !empty($context['validate_context']['code'])) {
       $this->code = $context['validate_context']['code'];
     }
-    $sid = variable_get('tfa_basic_twilio_account_sid', '');
-    $token = variable_get('tfa_basic_twilio_account_token', '');
+    $sid = \Drupal::config('tfa_basic.settings')->get('twilio_account_sid');
+    $token = \Drupal::config('tfa_basic.settings')->get('twilio_account_token');
```

Comment:
Does this account token / sid change by user? If so, is it a setting? Or, is it entered within a login form or added as a field to the user profile?

I want to make sure these new settings are not user specific. If so - they should not be settings